tests/unit/amo/pages/TestGuides.js

```diff
@@ -0,0 +1,108 @@
+import { oneLine } from 'common-tags';
+import * as React from 'react';
+
+import NotFound from 'amo/components/ErrorPage/NotFound';
+import HeadLinks from 'amo/components/HeadLinks';
+import { fetchGuidesAddons } from 'amo/reducers/guides';
+import {
+  dispatchClientMetadata,
+  fakeI18n,
+  shallowUntilTarget,
+} from 'tests/unit/helpers';
+import Guides, { extractId, GuidesBase, getContent } from 'amo/pages/Guides';
+
+describe(__filename, () => {
+  const getProps = ({
+    store = dispatchClientMetadata().store,
+    i18n = fakeI18n(),
+    dispatch = store.dispatch,
+    slug = 'privacy',
+    content = getContent('privacy', i18n),
+    match = {
+      params: {
+        slug,
+      },
+    },
+    ...customProps
+  } = {}) => {
+    return {
+      content,
+      dispatch,
+      i18n,
+      match,
+      slug,
+      store,
+      ...customProps,
+    };
+  };
+
+  const render = ({ ...customProps } = {}) => {
+    const allProps = getProps(customProps);
+
+    return shallowUntilTarget(<Guides {...allProps} />, GuidesBase);
+  };
+
+  it('fetches guides addons', () => {
+    const { store } = dispatchClientMetadata();
+    const dispatchSpy = sinon.spy(store, 'dispatch');
+    const root = render({ dispatch: dispatchSpy, store });
+
+    const content = getContent('privacy', fakeI18n());
+    const guids = content.sections.map((section) => section.addonGuid);
+
+    sinon.assert.calledWith(
+      dispatchSpy,
+      fetchGuidesAddons({
+        guids,
+        errorHandlerId: root.instance().props.errorHandler.id,
+      }),
+    );
+  });
+
+  it('renders a Guides Page', () => {
+    const slug = 'privacy';
+    const content = getContent(slug, fakeI18n());
+    const root = render({ content, slug });
+
+    expect(root.find('title')).toHaveText(content.title);
+
+    expect(root.find(HeadLinks)).toHaveLength(1);
+
+    expect(root.find('.Guides')).toHaveLength(1);
+    expect(root.find('.Guides-header-icon')).toHaveLength(1);
+    expect(root.find('.Guides-header-page-title')).toHaveLength(1);
+    expect(root.find('.Guides-header-intro')).toHaveLength(1);
+
+    expect(root.find('.Guides-section')).toHaveLength(1);
+    expect(root.find('.Guides-section-title').text()).toContain(
+      'Create and manage strong passwords',
```

Comment:
You may want to use `content.title`, no?

Reply:
I have to dig into the sections array for this which I can do..

Reply:
or did you mean the page title or the section title? or both? :)

for example the getContent will return this whole object here:
```javasscript
case 'privacy':
      return {
        title: i18n.gettext('Stay Safe Online'),
        introText: i18n.gettext(
          `The web is a wonderful but wild place. Your personal data can be used
           without your consent, your activities spied upon, and your passwords
           stolen. Fortunately, extensions can help fortify your online privacy
           and security.`,
        ),
        icon: 'stop-hand',
        sections: [
          // Bitwarden free password manager
          {
            addonGuid: '{446900e4-71c2-419f-a6a7-df9c091e268b}',
            header: i18n.gettext('Create and manage strong passwords'),
            description: i18n.gettext(
              `Password managers can help you create secure passwords, store your
               passwords (safely) in one place, and give you easy access to your
               login credentials wherever you are.`,
            ),
            addonCustomText: i18n.gettext(
              `Fully encrypted password protection. Store your data securely and
               access logins across devices.`,
            ),
            exploreMore: i18n.gettext(
              'Explore more %(linkStart)spassword manager%(linkEnd)s staff picks.',
            ),
            exploreUrl: '/collections/mozilla/password-managers/',
          },
        ],
      };
```